python/common_tool_methods.py

```diff
@@ -0,0 +1,146 @@
+import re
+import subprocess
+from datetime import datetime
+from typing import Dict, List
+import os
+
+from github import Repository, PullRequest
+from jinja2 import Environment, FileSystemLoader
+
+from .common_validations import *
+
+
+def get_spec_file_name(project_name: str) -> str:
+    return f"{project_name}.spec"
+
+
+def get_version_number(version: str, fancy: bool, fancy_release_count: int) -> str:
```

Comment:
We talked to @hanefi and we thought that we can remove it. Also for pg-auto-failover Dimitri didn't want to use fancy parameter. So to make it usable for all projects I made it parametric. We can discuss this again

Reply:
Seems that we don't even use `get_version_number` & `get_version_number_with_project_name` in nowhere, expect the unit-tests.
I think we should directly remove those functions.

Reply:
Fixed

Reply:
Let me make some corrections here:

PGAF already has fancy versioning. You can check out the package names. ( e.g. `-1` part in `1.5.2-1` in `postgresql-10-auto-failover-1.5_1.5.2-1_amd64.deb` )

What we discussed earlier was about the workflow parameter. Since a project will not stop using fancy versioning or start using it quite frequently, Jelte and I argued that it should be a hardcoded parameter.

We also argued that it is not necessary to have two separate variables here. However this is not as important. For example we can have a single variable `fancy_version` that when set to `-1` will imply that the project does not use fancy versioning.